src/users/users.resolver.ts

```diff
@@ -0,0 +1,24 @@
+import { Resolver, Mutation, Args, Query } from '@nestjs/graphql';
```

Comment:
Codacy has a fix for the issue: [Replace `Resolver,·Mutation,·Args,·Query` with `Args,·Mutation,·Query,·Resolver`](https://app.codacy.com/gh/uigormarshall/back-challenge-graphql/pullRequest?prid=8283306)
```suggestion
import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
```